### DISCUSSION TITLE
error running my api endpoint

### DISCUSSION THREAD

**Discussion by cometbus** [ayaka14732/ChatGPTAPIFree#14] [Q&A]
I've followed the steps outlined in this document. I think my api endpoint should look like this but it's not working

API_ENDPOINT = "https://chatgpt-api.429e4817165c49a4873cf8a012d97641.workers.dev/"

getting an error when I try to run commands though it

"root@DESKTOP-S70O6TN:~/openai-quickstart-python# curl -k https://chatgpt-api.429e4817165c49a4873cf8a012d97641.workers.dev/   -H 'Content-Type: application/json'   -d '{
>   "model": "gpt-3.5-turbo",
>   "messages": [{"role": "user", "content": "Hello, how are you?"}]
> }'
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Cache-Control" content="no-cache" />
<meta http-equiv="refresh" content="0; URL=/cgi-bin/index.cgi" />
</head>
<body style="background-color: white">
<a style="color: black; font-family: arial, helvetica, sans-serif;" href="/cgi-bin/index.cgi">Basic Router Info</a>
</body>
</html>"

**Comment by Sajjad0013** [comment]
Hi
